sdk/keyvault/azure-security-keyvault-certificates/test/ut/certificate_client_test.cpp

```diff
@@ -73,7 +71,7 @@ TEST_F(KeyVaultCertificateClientTest, CreateCertificateResumeToken)
     result = fromToken.PollUntilDone(m_defaultWait);
 
     auto cert = client.GetCertificate(certificateName);
-    EXPECT_EQ(cert.Value.Name(), options.Properties.Name);
+    //    EXPECT_EQ(cert.Value.Name(), options.Properties.Name);
```

Comment:
Can we fix it so that this line is not commented?

Reply:
done